src/sdl2_compat.c

```diff
@@ -3544,6 +3623,24 @@ SDL_Quit(void)
         GestureQuit();
     }
     SDL3_Quit();
+
+    if (joystick_list) {
+        SDL_free(joystick_list);
+        joystick_list = NULL;
+    }
+    num_joysticks = 0;
+
+    if (sensor_list) {
+        SDL_free(sensor_list);
```

Comment:
`SDL3_free` instead of `SDL_free` (same for the ones below)

Reply:
Fixed